pkg/reconciler/model.go

```diff
@@ -20,6 +20,11 @@ const (
 	Success    Status = "success"
 )
 
+const (
+	ManagedByLabel       = "managed-by"
+	LabelReconcilerValue = "reconciler"
```

Comment:
to make these labels unique to Kyma, it would be good to add a prefix like `reconciler.kyma-project.org/managed-by` or similar to them (do we have a common prefix we normally use in Kyma)?

Reply:
e.g. for serverless `serverless.kyma-project.io/managed-by` so changed this to `"reconciler.kyma-project.io/managed-by"`